src/controllers/auth.js

```diff
@@ -41,7 +41,7 @@ exports.login = async (req, res, next) => {
   }
   const token = jwt.sign(
     { _id: user._id.toHexString() },
-    process.env.JWT_SECRET,
+    process.env.SECRET_KEY,
```

Comment:
Thanks for fixing these, yet, I don't recommend you change such critical configurations at different branches. 
I mean they should be done at their own branches/commits.
(It's alright for our project)